hermes/probe/create_file.go

```diff
@@ -0,0 +1,114 @@
+package probe
+
+import (
+	"context"
+	"crypto/sha1"
+	"fmt"
+	"io"
+	"math/rand"
+	"os"
+	"sync"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/google/cloudprober/logger"
+	"github.com/google/cloudprober/metrics"
+	"github.com/google/cloudprober/probes/options"
+	"github.com/google/cloudprober/targets/endpoint"
+	"github.com/googleinterns/step224-2020/blob/evan/hermesMain/cmd"
+)
+
+type RandomFile struct {
+	Seed int64
+	Size int // File size in bytes
+}
+
+func (f *RandomFile) NewReader() *randomFileReader {
+	return &randomFileReader{size: f.Size, rand: rand.New(rand.NewSource(f.Seed))}
+}
+
+type randomFileReader struct {
+	size int
+	// Current reading index.
+	i    int
```

Comment:
nit: I would move this comment inline
```suggestion
	i    int  // Current reading index.
```